src/plugins/types.ts

```diff
@@ -10,6 +10,7 @@ export interface Plugin {
 
 export interface SlotMetaData {
   order?: number;
+  visibile?: (params: any) => boolean;
```

Comment:
```suggestion
  visible?: (params: any) => boolean;
```

Besides, do we need `params` there? These are declare in the `setup` function, which already has access to the `scope`:
```suggestion
  visible?: () => boolean;
```